From ca7013b61464c3516db1554534f6f01f2b6514ad Mon Sep 17 00:00:00 2001
From: "E. Madison Bray" <erik.bray@lri.fr>
Date: Wed, 8 Jan 2020 15:20:50 +0100
Subject: [PATCH] Trac #28695: These settings (if they are even needed at all)
 should use -print-prog-name, which on current gcc versions will provide the
 full path to the program.

---
 src/bin/sage-env | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/bin/sage-env b/src/bin/sage-env
index 49f29942df..c9623effc8 100644
--- a/src/bin/sage-env
+++ b/src/bin/sage-env
@@ -500,7 +500,7 @@ fi
 
 # Set AS to assembler used by $CC ("as" by default)
 if [ "$AS" = "" ]; then
-    CC_as=`$CC -print-file-name=as 2>/dev/null`
+    CC_as=`$CC -print-prog-name=as 2>/dev/null`
     if command -v $CC_as >/dev/null 2>/dev/null; then
         AS="$CC_as"
     fi
@@ -512,7 +512,7 @@ export AS
 
 # Set LD to linker used by $CC ("ld" by default)
 if [ "$LD" = "" ]; then
-    CC_ld=`$CC -print-file-name=ld 2>/dev/null`
+    CC_ld=`$CC -print-prog-name=ld 2>/dev/null`
     if command -v $CC_ld >/dev/null 2>/dev/null; then
         LD="$CC_ld"
     fi
-- 
2.21.0

